src/wp-includes/pluggable.php

```diff
@@ -693,7 +707,7 @@ function wp_validate_auth_cookie( $cookie = '', $scheme = '' ) {
 			return false;
 		}
 
-		$user = get_user_by( 'login', $username );
+		$user = get_user_by( 'login', $username, false );
```

Comment:
`get_user_by()` does not have a signature change, only the current user getter.

```suggestion
		$user = get_user_by( 'login', $username );
```